src/wallet/walletdb.cpp

```diff
@@ -57,6 +57,14 @@ bool CWalletDB::EraseTx(uint256 hash)
     return EraseIC(std::make_pair(std::string("tx"), hash));
 }
 
+bool CWalletDB::WriteKeyMeta(const CPubKey& vchPubKey, const CKeyMetadata& keyMeta)
+{
+    if (!batch.Write(std::make_pair(std::string("keymeta"), vchPubKey),
```

Comment:
Please add braces for readability

Should you be using `WriteIC` like everywhere else?